### DISCUSSION TITLE
<harmony> element

### DISCUSSION THREAD

**Discussion by tfakult35** [w3c-cg/musicxml#507] [Q&A]
I don't fully understand the description of the harmony element in the reference manual.

> "A <harmony> element can contain many stacked chords (e.g. V of II). Each individual chord including a required <kind> element is referred to as a harmony-chord. Stacked chords or secondary functions are represented using a sequence of harmony-chords. For example, V of II would be represented by a harmony-chord with a 5 numeral followed by a harmony-chord with a 2 numeral."

What is meant by stacked chords and secondary functions? My initial understanding was that secondary function refers to borrowed chords https://en.wikipedia.org/wiki/Borrowed_chord and the stacked chords refer to 2 or more chords atop each other played at the same time, creating more complex-sounding harmony. But these two things can't be represented the same way, right?

**Comment by mscuthbert** [comment]
Stacked chords are applied or secondary dominants, which are notated in roman numerals as a stack of independent chords.  My sense is to deemphasize the usage in the docs, since I've never heard the term outside of MusicXML, but if people know of other usages, let me know.
